dev/NavigationView/NavigationView.cpp

```diff
@@ -1115,6 +1119,51 @@ void NavigationView::UpdateIsClosedCompact()
     }
 }
 
+void NavigationView::UpdatePaneButtonsWidths()
+{
+    if (PaneDisplayMode() == winrt::NavigationViewPaneDisplayMode::Top
+        || DisplayMode() == winrt::NavigationViewDisplayMode::Minimal)
+    {
+        if (auto backButton = m_backButton.try_as<winrt::Button>())
+        {
+            backButton.Width(40);
+        }
+        if (auto paneToggleButton = m_paneToggleButton.try_as<winrt::Button>())
+        {
+            paneToggleButton.MinWidth(40);
+        }
+        if(auto paneButton = m_paneToggleButton.try_as<winrt::Button>())
```

Comment:
>if(auto paneButton = m_paneToggleButton.try_as<winrt::Button>()) [](start = 8, length = 64)

I think this could be combined with the if statement above

Reply:
Good catch, fixed. Thanks!